--- a/feeds/luci/modules/luci-mod-network/luasrc/model/cbi/admin_network/wifi.lua
+++ b/feeds/luci/modules/luci-mod-network/luasrc/model/cbi/admin_network/wifi.lua
@@ -258,6 +258,13 @@
 	s:taboption("advanced", Flag, "noscan", translate("Force 40MHz mode"),
 		translate("Always use 40MHz channels even if the secondary channel overlaps. Using this option does not comply with IEEE 802.11n-2009!")).optional = true
 
+	log_level = s:taboption("advanced", ListValue, "log_level", translate("Logging Level"))
+	log_level.optional = true
+	log_level:value("2", "Informational Message")
+	log_level:value("0", "Verbose Debugging")
+	log_level:value("1", "Debugging")
+	log_level:value("3", "Notification")
+	log_level:value("4", "Warning")
 	beacon_int = s:taboption("advanced", Value, "beacon_int", translate("Beacon Interval"))
 	beacon_int.optional = true
 	beacon_int.placeholder = 100
@@ -543,7 +550,7 @@
 
 	disassoc_low_ack = s:taboption("advanced", Flag, "disassoc_low_ack", translate("Disassociate On Low Acknowledgement"),
 		translate("Allow AP mode to disconnect STAs based on low ACK condition"))
-	disassoc_low_ack.default = disassoc_low_ack.enabled
+	disassoc_low_ack.default = disassoc_low_ack.disabled
 end
 
 
